app/src/rgb_underglow.c

```diff
@@ -299,6 +299,25 @@ int zmk_rgb_underglow_toggle() {
 
     state.on = !state.on;
 
+#if IS_ENABLED(CONFIG_ZMK_RGB_UNDERGLOW_EXT_POWER)
+    struct device *ext_power = device_get_binding("EXT_POWER");
```

Comment:
I would just initialize a static variable in the init once, instead of fetching each time.